addons/escoria-core/game/core-scripts/esc/commands/custom.gd

```diff
@@ -1,13 +1,15 @@
 # `custom object node func_name [params]`
 #
-# Calls the given Godot function on a (child) node of a registered `ESCitem`.
+# 
+# Executes the specified Godot function. This function must be in a script
+# attached to a child node of a registered `ESCitem`.
 #
 # **Parameters**
 #
 # - *object*: Global ID of the target `ESCItem`
 # - *node*: Name of the child node of the target `ESCItem`
 # - *func_name*: Name of the function to be called
-# - *params*: Any primitive, non-array arguments for the function. Multiple
+# - *params*: Any non-array arguments to be passed to the function. Multiple
```

Comment:
@dploeger : Comment from @BHSDuncan on this change - Do we allow for objects to be used as params @dploeger ? I thought we only allowed non-array primitive types?

Reply:
Yes. Agreed. No objects.

Reply:
So how would you like this reworded please @dploeger ?

Reply:
Hmmmm. Maybe like this?

```
# - *params*: Arguments to the function (array and object parameters are not supported). (...)
```

Reply:
"Any primitive, non-array argument" would include object, right? Or is an object non-primitive? Maybe. It's really nitpicky I'd say. :)

Reply:
primitive = non-object, non-array (usually native/intrinsic data types to the compiler, typically int, float, double, etc., and usually string, but I guess not always).

All good, though.

Reply:
That was the reason I removed the word "primative". I didn't think the term was accessible enough to the audience that they would be sure of the meaning. As the developers are confirming here exactly what it entails, it shows me it's not the right word to use - if it's not 100% immediately clear to you, then the users are going to be totally lost.

If you would like to leave the intent, can I suggest a variant of @dploeger's version from 9 hours ago? 
# - *params*:  Any arguments to be passed to the function (array and object parameters are not supported).

Reply:
I'm good with that.

I take your point; I sometimes forget that not everyone who's going to use this is a software engineer or someone who just inherently/intuitively knows what "primitive" means/represents.